README.md

```diff
@@ -112,15 +112,31 @@ Copy this value and add it to the file `functions/secrets.js`.
 const STRIPE_ENDPOINT_KEY = `${YOUR_LOCAL_SIGNING_SECRET_KEY}`;
 ```
 
-## Git flow
+## Our Git flow
+
+If you're new to Git / GitHub flows, then you may find these guides helpful:
+
+- https://guides.github.com/introduction/git-handbook/
+- https://guides.github.com/activities/forking/
+- https://guides.github.com/introduction/flow/
 
 To contribute to the code base, please follow these steps:
 
-- create a branch from `staging`
-- code
-- create a pull request on `staging`
+- fork the repository (note: Sparkle team skip this step)
+- create a new branch from `staging`
+- write your code
+- create a pull request to merge your branch into `staging`
+- wait for code review
+- fix any review comments
+- once the review has been finalised, a team member will **squash-merge** the PR into `staging`, which will trigger the CI to deploy the `staging` environment
+
+Then, to deploy to `production`:
 
-Then, to deploy to production, **merge staging into `master`**.
+- (a Sparkle team member will) create a PR to **merge staging into `master`** with a name such as **`deploy staging -> master`**
+- add the `🚀 deployment` label
+- copy the commit messages (including the `#1234` PR they were made in) and paste it into the PR description after `Deploys:` (see [example](https://github.com/sparkletown/sparkle/pull/1355))
+- **merge (not squash-merge)** this 'deployment PR' into `master`, this will trigger the CI system to deploy to the 'OG `production`' evironment
```

Comment:
```suggestion
- **merge (not squash-merge)** this 'deployment PR' into `master`, this will trigger the CI system to deploy to the '[OG](https://www.dictionary.com/e/slang/og/) `production`' environment
```